src/frontend/src/components/AppHeader/AppHeader.jsx

```diff
@@ -64,10 +75,34 @@ function AppHeader() {
                         <HeaderGlobalAction aria-label='Notifications'>
                             <Notification />
                         </HeaderGlobalAction>
-                        <HeaderGlobalAction aria-label={login}>
+                        <HeaderGlobalAction aria-label={login} onClick={togglePanel()}>
                             <UserAvatar />
                         </HeaderGlobalAction>
                     </HeaderGlobalBar>
+                    <HeaderPanel aria-label='Header Panel' expanded={isUserPanel}>
+                        <Switcher aria-label='Switcher Container'>
+                            <SwitcherItem isSelected aria-label='Profile' href='#'>
+                                Profile
+                            </SwitcherItem>
+                            <SwitcherDivider />
+                            <SwitcherItem href='#' aria-label='Link 2'>
+                                Manage Group Application
+                            </SwitcherItem>
+                            <SwitcherItem href='#' aria-label='Link 3'>
+                                Link 3
+                            </SwitcherItem>
+                            <SwitcherItem href='#' aria-label='Link 4'>
+                                Link 4
+                            </SwitcherItem>
+                            <SwitcherItem href='#' aria-label='Link 5'>
+                                Link 5
+                            </SwitcherItem>
+                            <SwitcherDivider />
+                            <SwitcherItem href='#' aria-label='Leave Group'>
+                                Leave Group
+                            </SwitcherItem>
```

Comment:
Let's keep this management central on the `MyGroup` page. This keeps our job simpler too, as we don't duplicate the Js

```suggestion
```